stable/kapacitor/values.yaml

```diff
@@ -3,7 +3,7 @@
 ##
 image:
   repository: "kapacitor"
-  tag: "1.2"
+  tag: "1.4-alpine"
```

Comment:
Why the switch to alpine?

Reply:
I added alpine to stay in line with the other images part of the stack. In charts, we are always using alpine. It is just for consistency.